lib/rex/post/meterpreter/ui/console/command_dispatcher/priv/elevate.rb

```diff
@@ -17,17 +17,20 @@ class Console::CommandDispatcher::Priv::Elevate
 
   include Console::CommandDispatcher
 
-  ELEVATE_TECHNIQUE_NONE					= -1
-  ELEVATE_TECHNIQUE_ANY					= 0
-  ELEVATE_TECHNIQUE_SERVICE_NAMEDPIPE		= 1
-  ELEVATE_TECHNIQUE_SERVICE_NAMEDPIPE2	= 2
-  ELEVATE_TECHNIQUE_SERVICE_TOKENDUP		= 3
-
-  ELEVATE_TECHNIQUE_DESCRIPTION = [ 	"All techniques available",
-                    "Service - Named Pipe Impersonation (In Memory/Admin)",
-                    "Service - Named Pipe Impersonation (Dropper/Admin)",
-                    "Service - Token Duplication (In Memory/Admin)"
-                  ]
+  ELEVATE_TECHNIQUE_NONE               = -1
+  ELEVATE_TECHNIQUE_ANY                = 0
+  ELEVATE_TECHNIQUE_SERVICE_NAMEDPIPE  = 1
+  ELEVATE_TECHNIQUE_SERVICE_NAMEDPIPE2 = 2
+  ELEVATE_TECHNIQUE_SERVICE_TOKENDUP   = 3
+
+  ELEVATE_TECHNIQUE_DESCRIPTION =
+    [
+      "All techniques available",
+      "Service - Named Pipe Impersonation (In Memory/Admin)",
+      "Service - Named Pipe Impersonation (Dropper/Admin)",
+      "Service - Token Duplication (In Memory/Admin)"
```

Comment:
If they're all services, we could probably remove `Service - ` from each element, and this would remove the need for the `dup` and `gsub` on line 57. Thoughts?